### PR TITLE
S17 committee apps

### PR DESCRIPTION
**Pull request by fat1996**

### CODE REVIEW

File: content/W2017/W17-new-committee-applications-S17.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+Title: Spring 2017 WiCS Committee Applications
```
Comment (annalorimer) [inline content/W2017/W17-new-committee-applications-S17.md]: The file name is a lil funky. To change just do "mv <oldfilename> <newfilename>"

File: content/W2017/W17-new-committee-applications-S17.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+Applications for the WiCS Undergraduate Committee for Spring 2017 are now open!
+We encourage you all to apply, since this is an excellent learning opportunity.
+You'll be doing things ranging from static website development to event 
+organization and linux systems administration.
+
+If you've applied previously, we encourage you to apply again!
+
+Kindly review the following guidelines before you apply:
+
++ You don't need any prior experience to apply. If you have a desire to improve the social and academic environment of women and non-binary people in computer science, that's exactly what we're looking for.
```
Comment (annalorimer) [inline content/W2017/W17-new-committee-applications-S17.md]: Lines should be 80 char.

**Review by annalorimer — CHANGES_REQUESTED** [review]

**Comment by annalorimer** [comment]
There's two files included in this pr ... You'll need to remove the one with the wrong name.

**Comment by fat1996** [comment]
@annalorimer deleted the extra file

**Review by annalorimer — APPROVED** [review]

**Pull request merged**